### ISSUE TITLE
✅ 🐞 Bug Report: TinyMCE toolbar buttons can't be visualized when selected when the editor is in Dark Mode - [ED-13471]

### ISSUE THREAD

**Issue by Joliamendi** [elementor/elementor#22642]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

If we click on any of its available options in the text editor widget, the color of the active button and icon will be white, and therefore the active button will no longer be recognizable. 

![text editor](https://github.com/elementor/elementor/assets/103376243/74a9f6a3-287c-435f-9d69-932abe63c519)

Maybe it would be better if the button icons are dark in active mode. Bottom image.

![better](https://github.com/elementor/elementor/assets/103376243/06bcb462-f890-471f-a26f-52f4ebd601c8)

### Steps to reproduce

Add the text editor widget and click on one of its buttons such as bold (B). After this option is activated, both the background color and the icon color will be white. Please note that this bug is only visible in **dark mode editor**.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


== Server Environment ==
Operating System: WINNT
Software: Apache/2.4.38 (Win32) OpenSSL/1.1.1a PHP/7.3.2
MySQL version: mariadb.org binary distribution v5.6.42-84.2
PHP Version: 7.3.2
PHP Memory Limit: 128M
PHP Max Input Vars: 1000
PHP Max Post Size: 64M
GD Installed: Yes
ZIP Installed: Yes
Write Permissions: All right
Elementor Library: Connected

== WordPress Environment ==
Version: 6.1.1
Site URL: https://domain.com/
Home URL: https://domain.com/
WP Multisite: No
Max Upload Size: 16 MB
Memory limit: 128M
Max Memory limit: 256M
Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
Language: en-US
Timezone: 0
Admin Email: [REDACTED]
Debug Mode: Inactive

== Theme ==
Name: Hello Elementor
Version: 2.7.0
Author: Elementor Team
Child Theme: No

== User ==
Role: administrator
WP Profile lang: en_US
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==
Elementor
Version: 3.14.0-beta1
Author: Elementor.com

== Elementor Experiments ==
Optimized DOM Output: Active
Improved Asset Loading: Active
Improved CSS Loading: Active
Inline Font Icons: Active
Accessibility Improvements: Inactive
Additional Custom Breakpoints: Inactive
admin_menu_rearrangement: Inactive
Flexbox Container: Active
Upgrade Swiper Library: Active
Grid Container: Active
Hello Theme Header & Footer: Inactive
Editor Top Bar: Active
Landing Pages: Inactive
Nested Elements: Active
Nested Accordion: Inactive by default
Lazy Load Background Images: Active
Global Style Guide: Inactive

````

</details>

**Comment by nicholaszein** [comment]
Hello @Joliamendi! 

Thank you for submitting your issue! 🙏 

As part of our continuing efforts to improve the Elementor Editor UI and Accessibility, we will discuss ways to improve it for future updates. Please keep in mind that for the time being we don't have an ETA for when this issue will be resolved, but rest assured we are aware of this issue and will be considering it.

Best regards

**Comment by nicholaszein** [comment]
Hello @Joliamendi!

I know I'm little late, but I'm just passing to let you know that...

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core v3.19.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
